### PR TITLE
bug/(origin-ui-core): Can't login after accepting organization invitation

### PR DESCRIPTION
**Pull request by JGiter**
Fixed: User with `Member` role can't login after accepting organization invitation

### CODE REVIEW

File: packages/origin-ui-core/src/features/users/sagas.ts [hunk 1]
```diff
@@ -91,7 +92,10 @@ function* fetchOffchainUserDetails(): SagaIterator {
 
             let organization: IOrganizationWithRelationsIds = null;
 
-            if (typeof userProfile.organization !== 'undefined') {
+            if (
+                typeof userProfile.organization !== 'undefined' &&
+                [Role.Admin || Role.SupportAgent].includes(userProfile.organization)
```
Comment (josipbagaric) [inline packages/origin-ui-core/src/features/users/sagas.ts]: @JGiter could you explain this line? I'm trying to understand what Roles have to do with the organization ID.

Comment (JGiter) [reply packages/origin-ui-core/src/features/users/sagas.ts]: In this block the user organization is fetched and if user doesn't have necessary rights the error is thrown

Comment (josipbagaric) [reply packages/origin-ui-core/src/features/users/sagas.ts]: Why are you comparing Roles to the Organization ID? The organization ID has no connection to Roles.
Roles are checked from `user.rights`, nothing to do with the organization.

You can also use the `isRole()` [helper function](https://github.com/energywebfoundation/origin/blob/1753e04781a422782d58874e8456e81d411cd460/packages/origin-backend-core/src/User.ts#L47).

`isRole(userProfile, Role.Admin, Role.SupportAgent)`.

Comment (JGiter) [reply packages/origin-ui-core/src/features/users/sagas.ts]: You right. Its typo

**Review by josipbagaric — CHANGES_REQUESTED** [review]
Please use the `isRole()` [helper function](https://github.com/energywebfoundation/origin/blob/1753e04781a422782d58874e8456e81d411cd460/packages/origin-backend-core/src/User.ts#L47) to minimize potential errors and make the code more maintainable and readable.

**Review by josipbagaric — APPROVED** [review]

**Pull request merged**